react/fields/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component, PropTypes } from 'react';
+import styles from './Checkbox.less';
+import CheckMarkIcon from '../../icons/CheckMarkIcon/CheckMarkIcon';
+import classnames from 'classnames';
+
+function combineClassNames(props = {}, ...classNames) {
+  const { className, ...restProps } = props;
+
+  return {
+    className: classnames.apply(null, [...classNames, className]), // eslint-disable-line no-useless-call
+    ...restProps
+  };
+}
+
+export default class Checkbox extends Component {
+  static displayName = 'Checkbox';
+
+  static propTypes = {
+    id: PropTypes.string.isRequired,
+    label: PropTypes.string.isRequired,
+    className: PropTypes.string,
+    inputProps: PropTypes.shape({
+      onChange: PropTypes.func,
+      checked: PropTypes.bool
+    })
+  }
+
+  static defaultProps = {
+    inputProps: {}
+  }
+
+  renderLabel() {
+    const { inputProps: { checked = false }, label, id } = this.props;
```

Comment:
You don't need to default `checked` to false as it will be falsy if it doesn't exist anyway. However, this expression will throw if `inputProps` is not supplied. I think we should make `inputProps` required and potentially `checked` as well.